gamemodes/terrortown/gamemode/client/cl_shopeditor.lua

```diff
@@ -26,7 +26,6 @@ function ShopEditor.GetEquipmentForRoleAll()
 
 	local eject = {
 		weapon_fists = true,
-		weapon_ttt_unarmed = true,
```

Comment:
Why would we want to do this exactly? 

I dont think we should allow someone to change this "weapon" because it is not really a weapon but rather a workaround and often used eg. by stuff like handcuffs etc.

Reply:
Yes please add it back

Reply:
done